.github/workflows/repackage.yml

```diff
@@ -117,11 +121,17 @@ jobs:
           # see .github/workflows/crawl.yml for details
           name: kernel-crawler-manifest
 
-      - name: Download Jobs and Cache
+      - name: Download Jobs
         uses: actions/download-artifact@v3
         with:
           name: package-jobs
 
+      - name: Download Jobs
+        uses: actions/download-artifact@v3
+        with:
+          name: package-cache
+          path: .build-data/cache/
```

Comment:
Am I seeing this right? Was the upload artifact not respecting the relative paths? That seems weird.